lnwallet/btcwallet/dropwtxmgr.go

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2015-2016 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+//
+// This file is a modified version of
+// https://github.com/btcsuite/btcwallet/blob/master/cmd/dropwtxmgr/main.go
+
+package btcwallet
+
+import (
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btclog"
+	"github.com/btcsuite/btcwallet/waddrmgr"
+	"github.com/btcsuite/btcwallet/wallet"
+	"github.com/btcsuite/btcwallet/walletdb"
+	"github.com/btcsuite/btcwallet/wtxmgr"
+)
+
+var (
+	// Namespace keys.
+	waddrmgrNamespace = []byte("waddrmgr")
+	wtxmgrNamespace   = []byte("wtxmgr")
+
+	// Bucket names.
+	bucketTxLabels = []byte("l")
+)
+
+func DropWalletTxManager(wallet *wallet.Wallet, log btclog.Logger) error {
```

Comment:
Thoughts on having this exported within the `btcwallet` repo so that we could just invoke it rather than maintaining two different copies?

Reply:
Yeah, that's the cleaner solution. Done: https://github.com/btcsuite/btcwallet/pull/723.